### PR TITLE
EIP-7251: Flatten get_active_balance

### PR DESCRIPTION
**Pull request by ppopth**
We have both get_active_balance and get_total_active_balance which have totally different meanings, since get_active_balance uses the balance field while get_total_active_balance uses the effective_balance field.

The names suggest that get_total_active_balance is the total of get_active_balance which is not true.

The name of get_active_balance doesn't quite make sense and it's used only in one place, so this commit flattens the logic of get_active_balance to the place where it's used.

### CODE REVIEW

**Review by mkalinin — APPROVED** [review]

File: specs/electra/beacon-chain.md [hunk 1]
```diff
@@ -884,10 +875,14 @@ def process_pending_consolidations(state: BeaconState) -> None:
 
         # Churn any target excess active balance of target and raise its max
         switch_to_compounding_validator(state, pending_consolidation.target_index)
+
+        # Calculate the consolidated balance
+        max_effective_balance = get_max_effective_balance(source_validator)
+        consolidated_balance = min(state.balances[pending_consolidation.source_index], max_effective_balance)
```
[the remark this answers is not in the document]
Comment (ppopth) [reply specs/electra/beacon-chain.md:881]: how about `source_balance`?

[the remark this answers is not in the document]
Comment (jtraglia) [reply specs/electra/beacon-chain.md:881]: Yes, that's a little better. I think `source_effective_balance` might be even better.

**Review by ralexstokes — APPROVED** [review]
weak +1 on Justin's comment, but otherwise love that we are shrinking the spec :)

**Review by jtraglia — APPROVED** [review]
Looks great. Thanks for making that change 😄

**Pull request merged**